fastlane/metadata/android/it/short_description.txt

```diff
@@ -0,0 +1 @@
+Goditi una schermata principale pulita mentre hai accesso ad ogni cosa in un istante
```

Comment:
This field is limited to 80 characters

Reply:
Edited in the previous comment. I paste it here too: "Goditi un'interfaccia pulita, mentre accedi ad ogni cosa in un istante!"

In screenshot 4 I translated as "permanent notification" because the rendition is much better.

Reply:
```suggestion
Goditi un'interfaccia pulita, mentre accedi ad ogni cosa in un istante!
```
Thank you!
Let's integrate it :-)